sample/SampleServer/Program.cs

```diff
@@ -40,15 +44,18 @@ static async Task MainAsync(string[] args)
                     .WithInput(Console.OpenStandardInput())
                     .WithOutput(Console.OpenStandardOutput())
                     .ConfigureLogging(x => x
-                        .AddSerilog()
+                        .AddSerilog(Log.Logger)
                         .AddLanguageProtocolLogging()
                         .SetMinimumLevel(LogLevel.Debug))
                     .WithHandler<TextDocumentHandler>()
                     .WithHandler<DidChangeWatchedFilesHandler>()
                     .WithHandler<FoldingRangeHandler>()
                     .WithHandler<MyWorkspaceSymbolsHandler>()
                     .WithHandler<MyDocumentSymbolHandler>()
-                    .WithHandler<SemanticTokens>()
+                    .WithHandler<SemanticTokensHandler>()
+                    // Linking here because the spec changed but vscode is behind
+                    .WithLink(TextDocumentNames.SemanticTokensFull, "textDocument/semanticTokens")
```

Comment:
@TylerLeonhardt this lets us fake out the vscode client for now, until it's updated to line up with the spec.